xonsh/procs/pipelines.py

```diff
@@ -331,6 +332,7 @@ def tee_stdout(self):
         enc = env.get("XONSH_ENCODING")
         err = env.get("XONSH_ENCODING_ERRORS")
         lines = self.lines
+        raw_lines = []
```

Comment:
This should probably be called `raw_out_lines`

Reply:
Done